### PR TITLE
#162948922 Article pagination

### PR DESCRIPTION
**Pull request by jamesbeamie**
#### What does this PR do?
Has the functionality for the user to view paginated articles

#### Description of Task(s) to be completed?
* Users should be able to view a specific number of articles per page, 
* The user can also choose the number of articles they want to view per page.

#### How should this be manually tested?
* Pull the branch `git fetch origin ft-pagination-162948922:ft-pagination-162948922`
* Checkout to the branch `ft-pagination-162948922`
* Install requirements `pip3 install -r requirements.txt`
* Run the server `python3 manage.py runserver`
* Launch postman and get all articles by sending a GET request to `http://127.0.0.1:8000/api/v1/articles/` 
* This gets the default ten articles per page.
* Get paginated articles by sending a GET request to `http://127.0.0.1:8000/api/v1/articles/?page_size=5` 
* This gets five articles per page
* To get articles on the next page, send a GET request to `http://127.0.0.1:8000/api/v1/articles/?page=2&page_size=5`
*This gets 5 articles from the next page
* Get paginated comments by sending a GET request to `http://127.0.0.1:8000/api/v1/articles/<slug>/comments/?page_size=15` 
*This gets 15 comments related to that specific article
* To get comments on the next page, send a GET request to `http://127.0.0.1:8000/api/v1/articles/<slug>/comments/?page=2&page_size=15`
*This gets 15 comments from page 2

#### What are the relevant pivotal tracker stories?
[#162948922](https://www.pivotaltracker.com/story/show/162948922)

### CODE REVIEW

File: authors/apps/core/pagination.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from rest_framework.pagination import PageNumberPagination
+
+class PaginateContent(PageNumberPagination):
+    """
+        Custmo pagination class
```
Comment (Ruiru11) [inline authors/apps/core/pagination.py]: You have a typo here

**Comment by Ruiru11** [comment]
You also have 3 commits squash them we agreed we should have one

**Review by boltonotieno — COMMENTED** [review]
When I try to fetch articles with a page_size of 5, it gets the default size which is 10. I'm I doing something wrong?

**Comment by boltonotieno** [comment]
Just realized I was doing the wrong thing. In the Url I appended a slash at the end like this
`http://127.0.0.1:8000/api/v1/articles/?page_size=5/` instead of  `http://127.0.0.1:8000/api/v1/articles/?page_size=5`

**Review by Ruiru11 — APPROVED** [review]

**Review by danuluma — APPROVED** [review]

**Review by frankip — COMMENTED** [review]
@jamesbeamie this is some nice work, have you included the pagination to the comments as well?

**Comment by boltonotieno** [comment]
You have 5 commits. Squash them into one commit message

**Review by boltonotieno — APPROVED** [review]

**Pull request merged**